README.md

```diff
@@ -3,7 +3,12 @@ Varnish HTTP accelerator container images
 
 [![Build and push images to Quay.io registry](https://github.com/sclorg/varnish-container/actions/workflows/build-and-push.yml/badge.svg)](https://github.com/sclorg/varnish-container/actions/workflows/build-and-push.yml)
 
-Varnish 6 status:[![Docker Repository on Quay](https://quay.io/repository/centos7/varnish-6-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/varnish-6-centos7)
+Images available on Quay are:
+* CentOS 7 [varnish-6](https://quay.io/repository/centos7/varnish-6-centos7)
```

Comment:
I see no builds available for varnish 6 either for fedora, Centos7, c8s, and c9s in quay.io. Is that intended?
Varnish5 centos7 build is 1 year old.

Reply:
Builds are available here: https://quay.io/repository/sclorg/varnish-6-c8s?tab=tags https://quay.io/repository/sclorg/varnish-6-c9s?tab=tags https://quay.io/repository/centos7/varnish-6-centos7?tab=tags

But varnish-5 can be removed because it is already deprecated.

I will remove the row.

Reply:
The builds are done by Quay.io itself. We are building it by ourselves.

Reply:
I see, thanks for the explanation.